artemis-server/src/main/java/org/apache/activemq/artemis/core/server/federation/FederatedQueueConsumerImpl.java

```diff
@@ -49,6 +55,8 @@ public class FederatedQueueConsumerImpl implements FederatedQueueConsumer, Sessi
    private final int intialConnectDelayMultiplier = 2;
    private final int intialConnectDelayMax = 30;
    private final ClientSessionCallback clientSessionCallback;
+   private volatile boolean started = false;
```

Comment:
The use of _started_ all looks synchronized, if so it doesnt need to be volatile (the earlier comment was only about _currentConnectTask_ which isnt all synced)